modules/health_quest/app/services/health_quest/configuration.rb

```diff
@@ -14,7 +14,9 @@ def service_name
     end
 
     def rsa_key
-      @key ||= OpenSSL::PKey::RSA.new(File.read(Settings.hqva_mobile.key_path))
+      private_key_path = Rails.env.development? ? Settings.hqva_mobile.local_key_path : Settings.hqva_mobile.key_path
```

Comment:
@ericboehs see: https://github.com/department-of-veterans-affairs/vets-api/pull/5202/files#r517545019